ert/data/__init__.py

```diff
@@ -4,10 +4,13 @@
 from ert.data._record import EnsembleRecord
 from ert.data._record import MultiEnsembleRecord
 from ert.data._record import RecordTransmitter
+from ert.data._record import RecordTransmitterType
 from ert.data._record import SharedDiskRecordTransmitter
 from ert.data._record import InMemoryRecordTransmitter
 from ert.data._record import RecordType
 from ert.data._record import RecordIndex
+from ert.data._record import record_data
+from ert.data._record import make_record
```

Comment:
I made the `make_record` function in the PR where I make the BlobRecord. At the time, I also made it global, so we would have a factory function for records. Markus thought it would be better not to do that and always be explicit about creating a `NumericalRecord`, or a `BlobRecord`, so I made it a private function. I see you use it once in `storage.py`. Do you think that call could be eleminated so we can make `make_record` private again?

Reply:
Not really possible unless I copy the logic it contains where I use it.